whois-query/src/test/resources/applicationContext-query-test-mock.xml

```diff
@@ -15,7 +15,7 @@
         <constructor-arg value="net.ripe.db.whois.query.handler.QueryHandler"/>
     </bean>
 
-    <bean id="accessControlListManager" class="org.mockito.Mockito" factory-method="mock" primary="true">
+    <bean id="AccessControlListManager" class="org.mockito.Mockito" factory-method="mock" primary="true">
```

Comment:
Use lowercase for the first word of the bean-id by convention, i.e. `accessControlListManager`